includes/widgets/social-icons.php

```diff
@@ -427,14 +446,30 @@ protected function register_controls() {
 			[
 				'label' => esc_html__( 'Spacing', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px', '%', 'em', 'rem' ],
+				'default' => [
+					'unit' => 'px',
+					'size' => 14,
```

Comment:
@rodolphebertozzo please change the size default to `'size' => 5,`

Reply:
Done @arielk !